src/applications/travel-pay/containers/TravelPayStatusApp.jsx

```diff
@@ -162,11 +163,12 @@ export default function App({ children }) {
 
   useEffect(
     () => {
-      if (userLoggedIn && travelClaims.length === 0) {
+      if (userLoggedIn && !hasFetchedClaims && travelClaims.length === 0) {
```

Comment:
leaving in `travelClaims.length === 0` because we still don't want extra fetches if claims are already fetched. Adding `hasFetchedClaims` lets us do this

Reply:
I have always heard that using `setState` in `useEffect` is problematic: https://stackoverflow.com/questions/53715465/can-i-set-state-inside-a-useeffect-hook

Reply:
@hinzed1127 any thoughts here?

Reply:
This should be ok because `hasFetchedClaims` is a simple flag and doesn't have any dependencies that would cause it to change and cause an infinite loop.

[The React docs go into more much detail](https://react.dev/learn/removing-effect-dependencies), but the main takeaway for our purposes here is that we're not dependent on any values that will get caught in an endless update loop